Pokedex_ios/Assets.xcassets/IconButtons/SnorlaxSelected.imageset/Contents.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
We're working with svg on all the images, please check with Yawar or Grover to know how they implemented the color change on their images without adding doubles

Reply:
fixed